fixtures/eslint/lib/cli-engine/file-enumerator.js

```diff
@@ -410,11 +410,10 @@ class FileEnumerator {
   /**
    * Iterate files in a given path.
    * @param {string} directoryPath The path to the target directory.
-   * @param {Object} options The options to iterate files.
+   * @param {object} options The options to iterate files.
    * @param {boolean} [options.dotfiles] If `true` then it doesn't skip dot files by default.
    * @param {boolean} [options.recursive] If `true` then it dives into sub directories.
    * @param {InstanceType<Minimatch>} [options.selector] The matcher to choose files.
-   * @returns {IterableIterator<FileEntry>} The found files.
```

Comment:
Should this be deleted or changed to `@yields`?

Reply:
Yep, fixed ✅